<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Proje Oluştur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Proje Oluştur&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="5">
      <widget class="QToolButton" name="dirB">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="4">
      <widget class="QLineEdit" name="classL">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="3">
      <widget class="QLineEdit" name="dirL">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="lbl5">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Kaynak Dosyası:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="3">
      <widget class="QLineEdit" name="sourceL"/>
     </item>
     <item row="11" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="10" column="1" colspan="5">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lbl2">
       <property name="text">
        <string>Sınıf İsmi:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="4">
      <widget class="QLineEdit" name="nameL">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>18</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="lbl4">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Proje Dizini:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="lbl3">
       <property name="text">
        <string>Pencere Türü:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>150</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbl1">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Proje İsmi:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="5">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>.py</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="4">
      <widget class="QComboBox" name="wtBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>QMainWindow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QWidget</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QDialog</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="lbl2_2">
       <property name="text">
        <string>Miras:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="4" colspan="2">
      <widget class="QPushButton" name="createB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Oluştur</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="4">
      <widget class="QLineEdit" name="instL"/>
     </item>
     <item row="11" column="3">
      <widget class="QPushButton" name="cancelB">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>İptal</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Python Sürümü:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="4">
      <widget class="QComboBox" name="versionB">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Python 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Python 2</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameL</tabstop>
  <tabstop>classL</tabstop>
  <tabstop>wtBox</tabstop>
  <tabstop>dirL</tabstop>
  <tabstop>dirB</tabstop>
  <tabstop>sourceL</tabstop>
  <tabstop>createB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelB</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
